py/asmthumb.c

```diff
@@ -36,6 +36,23 @@
 #include "py/mpstate.h"
 #include "py/asmthumb.h"
 
+#ifdef _MSC_VER
+#include <intrin.h>
+
+static uint32_t mp_clz(uint32_t x) {
+    unsigned long lz = 0;
+    return _BitScanReverse(&lz, x) ? 31 - lz : 0;
```

Comment:
`(sizeof(x) * 8 - 1) instead of `31` , or is that overkill ?

Reply:
done